src/components/Theme/Theme.js

```diff
@@ -0,0 +1,32 @@
+import React, { useContext } from "react"
+import PropTypes from "prop-types"
+import ThemeContext from "../../store/theme-context"
+
+const Theme = ({ children }) => {
+  const themeCtx = useContext(ThemeContext)
```

Comment:
щоб уникнути цієї проблеми неймінгу, можна одразу деструктурувати об'єкт

```suggestion
  const { name, toggleTheme } = useContext(ThemeContext)
```